docs-src/docs/setup/install.md

```diff
@@ -5,22 +5,25 @@
 Before you can install and configure Layer0, you must obtain the following:
 
 * **An AWS account.**
+Please have your AWS access key and secret ready.
+    - If the aws-cli is installed you can check them here `cat ~/.aws/credentials`
+    - If it's not installed assign them with `l0-setup init --aws-access-key <value> --aws-secret-key <value>`
```

Comment:
This addition seems unnecessary - the "Part 2: Create an Access Key#" section below covers this. also the 2nd bullet point doesn't make any sense.

Reply:
I agree it's out of context, but I think maybe it should be included in `## Part 3: Create a new Layer0 Instance`?  Or something along this line.